services/QuillLMS/spec/controllers/api/v1/classroom_units_controller_spec.rb

```diff
@@ -5,7 +5,8 @@
 describe Api::V1::ClassroomUnitsController, type: :controller do
   let(:other_teacher) { create(:teacher) }
   let(:classroom) { create(:classroom) }
-  let(:students) { create_list(:student, 5) }
+  let(:student_count) { 2 }
+  let(:students) { create_list(:student, student_count) }
```

Comment:
I also appreciate the extraction of the count into its own variable since it's used in comparisons below.